<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Cryptography</class>
 <widget class="QMainWindow" name="Cryptography">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>751</width>
    <height>354</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cryptography</string>
  </property>
  <widget class="QWidget" name="MainWindow">
   <property name="maximumSize">
    <size>
     <width>754</width>
     <height>16777215</height>
    </size>
   </property>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>751</width>
      <height>351</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="currentIndex">
     <number>4</number>
    </property>
    <widget class="QWidget" name="MainPage">
     <widget class="QPushButton" name="encryptText">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>90</y>
        <width>291</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Encrypt Text</string>
      </property>
     </widget>
     <widget class="QPushButton" name="decryptText">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>160</y>
        <width>291</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Decrypt Text</string>
      </property>
     </widget>
     <widget class="QPushButton" name="exit">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>230</y>
        <width>291</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="textEncrypt">
     <widget class="QPushButton" name="exitFromEncryptText">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>300</y>
        <width>80</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
     <widget class="QLabel" name="tobeEncryptedLabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>191</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Enter the Text to be Encrypted</string>
      </property>
     </widget>
     <widget class="QLabel" name="encryptedLabel">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>90</y>
        <width>181</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Encrypted Text</string>
      </property>
     </widget>
     <widget class="QPushButton" name="homeFromEncrypt">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>260</y>
        <width>80</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Home</string>
      </property>
     </widget>
     <widget class="QPushButton" name="encryptTextBtn">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>220</y>
        <width>80</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Encrypt</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="firstPrimeET">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>50</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="FirstPrimeLabelET">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>First Number</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="secondPrimeET">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>50</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="SecondPrimeLabelET">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>50</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Second Number</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="noticeEncryptedText">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>731</width>
        <height>31</height>
       </rect>
      </property>
      <property name="undoRedoEnabled">
       <bool>false</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="plainText">
       <string>Please remember the Private key that appears after the Encryption Process inorder to Decrypt the Data.</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="tobeEncryptedText">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>291</width>
        <height>231</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="encryptedText">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>110</y>
        <width>291</width>
        <height>231</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="textDecrypt">
     <widget class="QPushButton" name="exitFromDecryptText">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>300</y>
        <width>80</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="DecryptedText">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>291</width>
        <height>231</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="EncryptedText">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>110</y>
        <width>281</width>
        <height>231</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="homeFromDecrypt">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>260</y>
        <width>80</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Home</string>
      </property>
     </widget>
     <widget class="QPushButton" name="DecrptTextBtn">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>220</y>
        <width>80</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Decrypt</string>
      </property>
     </widget>
     <widget class="QLabel" name="decryptedLabel">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>90</y>
        <width>181</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Decrypted Text</string>
      </property>
     </widget>
     <widget class="QLabel" name="tobeDecryptedLabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>191</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Enter the Text to be Decrypted</string>
      </property>
     </widget>
     <widget class="QLabel" name="FirstPrimeLabelDT">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Please Enter the Private Key</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="privateKeyText">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>30</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="fileEncrypt">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QWidget" name="fileDecrypt">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>exit</sender>
   <signal>clicked()</signal>
   <receiver>Cryptography</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>378</x>
     <y>242</y>
    </hint>
    <hint type="destinationlabel">
     <x>403</x>
     <y>357</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
